.github/workflows/selfbuild.yml

```diff
@@ -39,7 +39,7 @@ jobs:
             curl \
               --user-agent "cargo-quickinstall build pipeline (alsuren@gmail.com)" \
               --fail "https://crates.io/api/v1/crates/cargo-quickinstall" \
-              | jq -r .versions[0].num
+              | jq -r '.versions[0].num'
           )
           env VERSION=$VERSION TARGET_ARCH=$TARGET_ARCH ./build-version.sh cargo-quickinstall
```

Comment:
```suggestion
         VERSION="$VERSION" TARGET_ARCH="$TARGET_ARCH" ./build-version.sh cargo-quickinstall
```

Reply:
heh. I can't remember what got me into the habit of writing `env` explicitly. It might be from writing portable npm scripts for running on windows or something.